jekyll/_cci2/server-3-install-migration.adoc

```diff
@@ -84,6 +87,9 @@ notify your team of the new CircleCI UI and URL, if it has changed.
 ### Why does nothing happen when I select "Start Building" on my project after migration?
 * By default, a newly added project (a project that has never been followed) will trigger a build automatically after it has been followed for the first time. If the project was or ever has been followed in 2.0 or 3.0, it will not be considered a new project or first build and a build will not be triggered after follow. To trigger a build, perform an activity that will trigger a Github webhook such as pushing up a new commit or branch.
 
+### I got an error "Error from server (NotFound):" 
+* The script assumes a specific naming patterns for your Postgres and MongoDB. If you get this error, it may indicate a non-standard installation, a missing DB migration, or other issues. In this case it is best to contact support with a support bundle and the output from the migration script. 
```

Comment:
```suggestion
* The script assumes specific naming patterns for your Postgres and MongoDB. If you get this error, it may indicate a non-standard installation, a missing DB migration, or other issues. In this case it is best to contact support with a support bundle and the output from the migration script. 
```